### ISSUE TITLE
detect-only mode

### ISSUE THREAD

**Issue by eadmaster** [shhavel/facerec#2]
is it possible to only detect the faces inside an image?
(like [facedetect](https://www.thregr.org/~wavexx/software/facedetect/) does)

**Comment by shhavel** [comment]
@eadmaster Yes, this micro lib is only for images. Face recognition in videos implemented in OpenCV, but I didn't investigate it. Thanks.

**Comment by eadmaster** [comment]
I meant face **detection** for static images, not videos...
(detection = find the rectangle coordinates containing the faces).
e.g.:
```
./facerec detect path/to/image.jpg
289 139 56 56 # first face found
295 283 55 55 # second face found
```

**Comment by shhavel** [comment]
Currently yes - only faces.
Object types declared in [PROFILES](https://github.com/shhavel/facerec/blob/master/facerec#L13)
Maybe the same algorithm can be used to detect different types of objects - I don't know.
